azure-pipelines.yml

```diff
@@ -2,6 +2,7 @@ name: $(BuildDefinitionName)_$(Date:yyyyMMdd)$(Rev:.rr)
 variables:
   CIBW_MANYLINUX_X86_64_IMAGE: manylinux2014
   CIBW_MANYLINUX_I686_IMAGE: manylinux2014
+  JSOC_EMAIL: "jsoc@cadair.com"  # For the example gallery on the dev deps build
```

Comment:
We are dropping this on the metadata PR.

Reply:
Dropping what?

Reply:
The gallery build.

Reply:
Well we can pull this in that PR if this gets merged first?

Reply:
Fair.